docs/app/pages/Configuration.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <page-container centered :title="$t('pages.configuration.title')">
+    <div class="page-container-section">
+      <h2 class="md-headline">How to customized</h2>
+
+      <p>Vue Material provide some options for customized. These options are reactive, you can change it anytime, anywhere:</p>
+      <code-example :title="$t('pages.configuration.viaVueGlobalObject')">
+        import Vue from 'vue'
+
+        // change single option
+        Vue.material.locale.dateFormat = 'DD/MM/YYYY'
+
+        // change multiple options
+        Vue.material = {
+          ...Vue.material,
+          locale: {
+            ...Vue.material.locale,
+            dateFormat: 'DD/MM/YYYY',
+            firstDayOfAWeek: 1
+          }
+        }
+      </code-example>
+      <p>or you can change it via <code>this.$material</code> in a vue component:</p>
+      <code-example :title="$t('pages.configuration.inVueComponents')">
+        export default {
```

Comment:
This can also be a table with the API. What do you think?

Reply:
I don't know. The configuration object is nested. I'm afraid that table can't describe its structure clearly.